packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts

```diff
@@ -90,7 +103,7 @@ export const generateAmplifyOverrideProjectBuildFiles = (backendDir: string, src
   }
```

Comment:
Can we remove addition of tsconfig at the prooject level?

Reply:
Do you mean to remove tsConfig from everwhere ?